.eslintrc.js

```diff
@@ -473,5 +473,36 @@ module.exports = {
         'import/prefer-default-export': 'off',
       },
     },
+    {
+      /**
+       * Examples are for demonstration purposes and should not be considered a part of the library.
```

Comment:
@Janpot we will be able to remove https://github.com/mui/mui-toolpad/blob/282d1b72b789048df6a98d2a2a6e48c4b77a0bd9/.eslintrc.js#L115